src/plugins/mysql.js

```diff
@@ -34,6 +34,7 @@ function createWrapQuery (tracer, config) {
       if (sequence._callback) {
         sequence._callback = wrapCallback(tracer, span, childOf, sequence._callback)
       } else {
+        scope.bind(sequence)
```

Comment:
Is `sequence` always an event emitter even when `sequence._callback` is available? If that's the case, it should probably be bound outside of the condition.